Sources/BazelizeKit/Codegen/Codegen+Application.swift

```diff
@@ -7,20 +7,24 @@
 
 import Foundation
 import XCode
+import PluginInterface
 
 extension Target {
+    #warning("todo config")
+    var setting: XCodeBuildSetting { self["Release"]! }
+    
     func generateApplicationCode(_ kit: Kit) -> String {
-        precondition(self.sdk != nil, "sdk")
+        precondition(setting.sdk != nil, "sdk")
```

Comment:
(not required for this PR)
In the future, we may need to throw Error when `sdk` is `nil`.

`precondition` won’t be evaluated for `Release` builds